api/src/pcapi/routes/backoffice_v3/accounts.py

```diff
@@ -236,6 +249,551 @@ def render_public_account_details(
     )
 
 
+class TunnelType(enum.Enum):
+    NOT_ELIGIBLE = "not-eligible"
+    AGE18 = EligibilityType.AGE18.value
+    UNDERAGE = EligibilityType.UNDERAGE.value
+    UNDERAGE_AGE18 = f"{EligibilityType.UNDERAGE.value}+{EligibilityType.AGE18.value}"
+
+
+def _get_tunnel_type(user: users_models.User) -> TunnelType:
+    if user.birth_date is None:
+        return TunnelType.NOT_ELIGIBLE
+
+    age_at_creation = users_utils.get_age_at_date(user.birth_date, user.dateCreated)
+    age_now = users_utils.get_age_from_birth_date(user.birth_date)
+
+    if age_now < users_constants.ELIGIBILITY_AGE_18:
+        return TunnelType.UNDERAGE
+    if age_at_creation < users_constants.ELIGIBILITY_AGE_18 <= age_now:
+        return TunnelType.UNDERAGE_AGE18
+    if age_at_creation == users_constants.ELIGIBILITY_AGE_18:
+        return TunnelType.AGE18
+    return TunnelType.NOT_ELIGIBLE
+
+
+def _get_status_success(status: str) -> bool:
+    return status in (
+        SubscriptionItemStatus.OK.value,
+        SubscriptionItemStatus.NOT_APPLICABLE.value,
+        SubscriptionItemStatus.NOT_ENABLED.value,
+        SubscriptionItemStatus.SKIPPED.value,
+    )
+
+
+def _get_status_error(status: str) -> bool:
+    return status == SubscriptionItemStatus.KO.value
+
+
+def _get_status_warning(status: str) -> bool:
+    return status in (SubscriptionItemStatus.PENDING.value, SubscriptionItemStatus.SUSPICIOUS.value)
+
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**🎯 suggestion:** ‏
```python
def _get_status(status: str) -> str: 
    if status in (SubscriptionItemStatus.OK.value, SubscriptionItemStatus.NOT_APPLICABLE.value, SubscriptionItemStatus.NOT_ENABLED.value, SubscriptionItemStatus.SKIPPED.value): 
        return 'sucess'

    if status == SubscriptionItemStatus.KO.value:
        return 'error'
    
    if status in (SubscriptionItemStatus.PENDING.value, SubscriptionItemStatus.SUSPICIOUS.value):
        return warning
```

Si on veut absolument `status` de la forme 
```python
            status={
                "error": bool,
                "success": bool,
                "warning": bool,
                "active": bool,
                "disabled": bool,
            },
``` 
on peut mettre à jour le init de la classe `RegistrationStep`